examples/page-location/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+
+import { ContentType, PageExtensionSDK } from 'contentful-ui-extensions-sdk';
+import { TabPanel, Heading, Paragraph } from '@contentful/forma-36-react-components';
+
+import Collection from './Collection';
+import CollectionList from './CollectionList';
+
+interface DashboardProps {
+  sdk: PageExtensionSDK;
+  contentTypes: ContentType[];
+}
+
+interface CollectionsState {
+  total: number | null;
+  published: number | null;
+  scheduled: number | null;
+  recent: any[] | null;
+}
+
+export default function Dashboard({ sdk, contentTypes }: DashboardProps) {
+  const [data, setData] = useState<CollectionsState>({
+    total: null,
+    published: null,
+    scheduled: null,
+    recent: null
+  });
+
+  useEffect(() => {
+    async function fetchData() {
+      // Fetch some basic statistics.
+      const total = await sdk.space
+        .getEntries()
+        .then(entries => entries.total)
+        .catch(() => 0);
+      const published = await sdk.space
+        .getPublishedEntries()
+        .then(entries => entries.total)
+        .catch(() => 0);
+      const scheduled = await sdk.space
+        .getAllScheduledActions()
+        .then(entries => entries.length)
+        .catch(() => 0);
```

Comment:
Optional: I would use a `Promise.all` that way this makes all the requests at the same time.